src/documents/models.py

```diff
@@ -463,3 +463,17 @@ def from_filename(cls, filename) -> "FileInfo":
                 cls._mangle_property(properties, "created")
                 cls._mangle_property(properties, "title")
                 return cls(**properties)
+
+
+# Extending User Model Using a One-To-One Link
+class UiSettings(models.Model):
```

Comment:
I think it would be a good enhancement to include this model in the export tool.

Reply:
Sure! How do I do that? 😜
(Or do you think should be separate PR)

Reply:
It should be pretty easy (famous last words).  In `document_exporter`, it's step 2.

Adding something like `manifest += json.loads(serializers.serialize("json", UiSettings.objects.all()))` should be sufficient (after importing the model)

Reply:
Awesome, I added this and tested export + import, easy!